tests/data_association/test_point3d_initializer.py

```diff
@@ -147,10 +147,8 @@ def testSimpleTriangulationWithOneMeasurement(self):
         self.assertTrue(self.__runWithOneMeasurement(self.simple_triangulation_initializer))
 
     def testSimpleTriangulationWithOutlierMeasurements(self):
-
         sfm_track, _, _ = self.simple_triangulation_initializer.triangulate(SfmTrack2d(get_track_with_one_outlier()))
-
-        self.assertIsNone(sfm_track)
+        assert sfm_track.number_measurements() == 7
```

Comment:
thanks for the PR, Travis. Could we add an inline comment explaining why we expect 7 tracks here?

Reply:
Done